srcStatic/OP2Memory.cpp

```diff
@@ -27,97 +27,97 @@ bool EnableOp2MemoryPatching()
 
 	// Enable memory patching for Outpost2.exe, and set relocation offset
 	memoryPatchingEnabled = true;
-	loadOffset = reinterpret_cast<std::uintptr_t>(op2ModuleBase) - ExpectedOutpost2Addr;
+	loadOffset = reinterpret_cast<std::uintptr_t>(op2ModuleBaseAddress) - ExpectedOutpost2Address;
 	return true;
 }
 
 
+bool Op2UnprotectMemory(std::uintptr_t destBaseAddress, std::size_t size)
+{
+	if (!memoryPatchingEnabled) {
+		return false;
+	}
+
+	void* destAddress = reinterpret_cast<void*>(destBaseAddress + loadOffset);
+	// Try to unprotect memory
+	DWORD oldAttr;
```

Comment:
Good points.

I noticed the `base` abbreviation just as I was writing up the PR. Was tempted to go back and change it, but thought I might be losing the focus of the PR. I may update that in the future. Maybe "target" rather than "destination"?